creativecloud/deps/merch-sidenav.js

```diff
@@ -1,9 +1,9 @@
-// branch: catalog-regressions-4 commit: 90641e42ad012c2386133cbd2df855f468842b2d Wed, 29 May 2024 12:05:08 GMT
+// branch: main commit: dd5f72e1449ed667f48ef5be086043ddf9cc113d Mon, 24 Jun 2024 12:52:40 GMT
```

Comment:
can you re-build from [ecf0570](https://github.com/adobecom/mas/commit/ecf0570abe50b3871d2e7b823cbdbdcc11f3e133) ?